api/TouchEvent.json

```diff
@@ -0,0 +1,522 @@
+{
+  "api": {
+    "TouchEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/TouchEvent",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": "22"
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true,
+            "flags": [
+              {
+                "type": "preference",
+                "name": "Standards Preview",
+                "value_to_set": "true"
+              }
+            ]
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": [
```

Comment:
Small thing - I'd be tempted to supplement this representation with a `notes` field just so folk can tell why support was dropped and re-added.

Reply:
I added the explanation on the basic support only. I thought it's most relevant there and it would avoid duplication (for the reader).